32blit-stm32/Src/fatfs_sd.c

```diff
@@ -559,3 +540,577 @@ DRESULT SD_disk_ioctl(BYTE drv, BYTE ctrl, void *buff)
 
 	return res;
 }
+

```

Comment:
Probably don't want the entire old file as a comment

Reply:
Whoops!